### PR TITLE
fix several typos

### PR DESCRIPTION
**Pull request by TJ-91**

### CODE REVIEW

**Review by falko-strenzke — APPROVED** [review]

File: draft-ehlen-openpgp-nist-bp-comp.md
Comment (sehlen) [inline draft-ehlen-openpgp-nist-bp-comp.md:566]:
```suggestion
 - The ML-KEM encapsulation algorithm is invoked to create an ML-KEM ciphertext together with an ML-KEM shared secret key.
```

The usual KEM terminolofy is "shared secret" and in FIPS 203 ipd they write "share secret key".

Comment (sehlen) [reply draft-ehlen-openpgp-nist-bp-comp.md:566]: This might have to be applied to other places as well. I don't object to leaving it as is, but I think "shared secret (key)" would be better.

**Review by sehlen — APPROVED** [review]
This is fine to merge, if you don"t agree with my suggestion, this can be made an additional issue.

File: draft-ehlen-openpgp-nist-bp-comp.md
Comment (falko-strenzke) [reply draft-ehlen-openpgp-nist-bp-comp.md:566]: In order not to hold up our submission version, I made an issue for your proposal: https://github.com/openpgp-pqc/draft-ehlen-openpgp-nist-bp-comp/issues/11

**Pull request merged**
